prow/cmd/gerrit/main.go

```diff
@@ -41,14 +41,27 @@ import (
 )
 
 type options struct {
+	// gcsCredentialsFile string is used for reading/writing to GCS block storage.
+	// If you want to write to "file://" paths, this parameter is optional.
+	// If set, this file is used to read/write to gs:// paths
+	// If not, go cloud credential auto-discovery is used
+	// For more details see the pkg/io/v2/providers pkg.
 	gcsCredentialsFile string
-	cookiefilePath     string
-	configPath         string
-	jobConfigPath      string
-	projects           client.ProjectsFlag
-	lastSyncFallback   string
-	dryRun             bool
-	kubernetes         prowflagutil.KubernetesOptions
+	// 	s3CredentialsFile string is used for reading/writing to s3 block storage.
+	// If you want to write to "file://" paths, this parameter is optional.
+	// If set, this file is used to read/write to s3:// paths
+	// If not, go cloud credential auto-discovery is used
+	// For more details see the pkg/io/v2/providers pkg.
+	s3CredentialsFile string
+	cookiefilePath    string
+	configPath        string
+	jobConfigPath     string
+	projects          client.ProjectsFlag
+	// lastSyncFallback is the path to sync the latest timestamp
+	// Can be file:///local/path, gs://path/to/object or s3://path/to/object.
```

Comment:
Can we change the docs back to `/local/path`?

Reply:
Done